src/devices/vdml_usd.cpp

```diff
@@ -43,6 +43,19 @@ std::vector<std::string> list_files(const char* path) {
 			return files;
 		}
 	}
+
+	// Check path user passed in
+	std::string_view path_sv(path);
+	size_t found = path_sv.find("usd");
+	if (found == 0 || found == 1) {
+		// Deal with when user prepends path with usd
+		// as either "usd/..." or "/usd/..."
+		path_sv.remove_prefix(3);
+	}
```

Comment:
How does hardcoding the `3` here deal with both cases? Wouldn't it be `3 + found`?

Also to avoid the magic-number-ness of this 3, I'd probably write this as:
```cpp
constexpr std::string_view usd_prefix {"usd"};
std::string_view path_sv {path};
const size_t usd_prefix_idx = path_sv.find(usd_prefix);
if (found == 0 || found == 1) {
    path_sv.remove_prefix(usd_prefix.length() + usd_prefix_idx);
}
```

Reply:
Also what happens here if they don't include the usd prefix? We just make it work the same as if they did? May be weird, since we also always _return_ paths prefixed with `/usd/`.

My preference here would be to return a "path not found" error, as just assuming they wanted an implicit `/usd/` makes doing anything with a VFS outside of `/usd/` in the future into a backwards compatibility break.